src/mon/AuthMonitor.cc

```diff
@@ -803,6 +807,20 @@ bool AuthMonitor::preprocess_command(MonOpRequestRef op)
   cmd_getval(g_ceph_context, cmdmap, "format", format, string("plain"));
   boost::scoped_ptr<Formatter> f(Formatter::create(format));
 
+
+  list<AuthMonReadCommandRef> read_cmds = {
+    AuthMonReadCommandRef(new AuthMonAuthExport(mon, this, g_ceph_context))
+  };
+
+  for (auto &cmd : read_cmds) {
```

Comment:
This looks fine to me, but if we are looking to generalize it perhaps it belongs in its own function?